target_position: [0, 0, 1]
